server/nodeset/upload-deposit-data.go

```diff
@@ -7,18 +7,23 @@ import (
 	"net/url"
 
 	"github.com/goccy/go-json"
+	eth2types "github.com/wealdtech/go-eth2-types/v2"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/gorilla/mux"
+	swcommon "github.com/nodeset-org/hyperdrive-stakewise/common"
+
 	duserver "github.com/nodeset-org/hyperdrive-daemon/module-utils/server"
 	swapi "github.com/nodeset-org/hyperdrive-stakewise/shared/api"
-	"github.com/rocket-pool/node-manager-core/api/server"
 	"github.com/rocket-pool/node-manager-core/api/types"
 	"github.com/rocket-pool/node-manager-core/beacon"
 	"github.com/rocket-pool/node-manager-core/eth"
-	"github.com/rocket-pool/node-manager-core/utils/input"
 	"github.com/rocket-pool/node-manager-core/wallet"
-	eth2types "github.com/wealdtech/go-eth2-types/v2"
+)
+
+const (
+	pendingState         = "PENDING"
```

Comment:
When you're defining constants you should make the type explicit, like `pendingState string = "PENDING"`
Also since this is a constant sent by nodeset.io, you should probably put it in nodeset-client instead of here.

Reply:
resolved